src/components/local-storage-toggle.tsx

```diff
@@ -28,11 +37,40 @@ export const LocalStorageToggle: React.FC<LocalStorageToggleProps> = ({ resetKey
   }
 
   return (
-    <div {...props} className={`${props.className} local-storage-toggle input-reset bn black-80 w-100 mb3`}>
-      <input className="status" style={{ display: 'none' }} id={localStorageKey} type="checkbox" name="status" checked={isChecked} onChange={handleChange} />
-      <label htmlFor={localStorageKey} className="w-100 h-100">
-        <div className="status-switch relative h3-custom pointer white bg-gray w-100" data-unchecked={offLabel} data-checked={onLabel} />
+    <div {...props} className={`${props.className}`}>
+      <span className='f5 ma0 pt3 teal fw4 db'>{label}</span>
+      {(description.length > 0) ? <span className="charcoal-muted f6 fw1 db pt1 lh-copy mb2">{description}</span> : null}
+
+      <input
+        type="checkbox"
+        id={localStorageKey}
+        checked={isChecked}
+        onChange={handleChange}
+        className="dn"
+      />
+      <label
+        htmlFor={localStorageKey}
+        className={`relative dib h2 w3 flex-shrink-0 pointer br4 ${
+          isChecked ? 'bg-green' : 'bg-dark-gray'
+        } transition-all duration-200 ease-in-out`}
+      >
+        <span
+          className={`absolute top-0 left-0 dib h2 w2 br-100 bg-white shadow-1 ${
+            isChecked ? 'translate-x-100' : 'translate-x-0'
+          } transition-transform duration-200 ease-in-out`}
+        />
       </label>
```

Comment:
I think we should reduce the size of this toggle, it looks a little too large right now?

changing the size from `h2 w2` to `h1 w1` (for the span) and from `h2 w3` to `h1 w2` (for the label) looks a little more appealing to me.. what do you think?

<img width="320" alt="image" src="https://github.com/user-attachments/assets/95925304-db5a-473e-ad53-bc2c124478ae">

Reply:
Also, do we need to change the styles of these toggles? we had a review of the toggle UI a while back

Reply:
The old toggle was constantly confusing me.
I don't think thing having text helps, because you can still read the text of the disabled state:


![Screenshot 2024-10-22 at 6 01 47 PM](https://github.com/user-attachments/assets/876b8ae1-1e8a-4a99-9c70-07087adb4545)
![Screenshot 2024-10-22 at 6 02 45 PM](https://github.com/user-attachments/assets/2924570e-43fa-42ae-84e4-0bd52be626d6)

Reply:
Binary toggles with a label that describes their behaviour in binary terms are more easily understandable.

As for the size. I think it's a good size currently, more or less the same as the old toggle and the save button.

Reply:
ya you're right. I do like the new toggle better. what do you think about making it slightly smaller?

Reply:
Yeah I'll try making it smaller

Reply:
>changing the size from h2 w2 to h1 w1 (for the span) and from h2 w3 to h1 w2 (for the label) looks a little more appealing to me.. what do you think?


Seemed a little too small to my taste, but they're about the same size as the built in macOS toggles, so I'll defer to Apple's design rigour  😅
![Screenshot 2024-10-23 at 4 28 01 PM](https://github.com/user-attachments/assets/a6bb9945-132c-427a-bf89-5578d6c6da00)